### PR TITLE
Created RPM package spec file to generate RPM package from the source.

### PR DESCRIPTION
**Pull request by jokLiu**
Tested locally that the generation of RPM package works and the the required commands are properly installed:
```
vagrant@ubuntu-xenial:~/repos/nvdtools$ sudo alien -i /home/vagrant/rpmbuild/RPMS/x86_64/nvdtools-1.0-1.x86_64.rpm
	dpkg --no-force-overwrite -i nvdtools_1.0-2_amd64.deb
(Reading database ... 74500 files and directories currently installed.)
Preparing to unpack nvdtools_1.0-2_amd64.deb ...
Unpacking nvdtools (1.0-2) over (1.0-2) ...
Setting up nvdtools (1.0-2) ...
vagrant@ubuntu-xenial:~/repos/nvdtools$ rpmbuild -ba nvdtools.spec
Executing(%prep): /bin/sh -e /var/tmp/rpm-tmp.m2ixrA
+ umask 022
+ cd /home/vagrant/rpmbuild/BUILD
+ cd /home/vagrant/rpmbuild/BUILD
+ rm -rf nvdtools-1.0
+ + /bin/tar -xf -
/bin/gzip -dc /home/vagrant/rpmbuild/SOURCES/nvdtools-1.0.tar.gz
+ STATUS=0
+ [ 0 -ne 0 ]
+ cd nvdtools-1.0
+ /bin/chmod -Rf a+rX,u+w,g-w,o-w .
+ exit 0
Executing(%build): /bin/sh -e /var/tmp/rpm-tmp.cRnGx9
+ umask 022
+ cd /home/vagrant/rpmbuild/BUILD
+ cd nvdtools-1.0
+ go build -o cpe2cve ./cmd/cpe2cve/cpe2cve.go
+ go build -o csv2cpe ./cmd/csv2cpe/csv2cpe.go
+ go build -o nvdsync ./cmd/nvdsync/main.go
+ go build -o rpm2cpe ./cmd/rpm2cpe/rpm2cpe.go
+ exit 0
Executing(%install): /bin/sh -e /var/tmp/rpm-tmp.MWbG8N
+ umask 022
+ cd /home/vagrant/rpmbuild/BUILD
+ cd nvdtools-1.0
+ install -d /home/vagrant/rpmbuild/BUILDROOT/nvdtools-1.0-1.x86_64/usr/bin
+ install -p -m 0755 ./cpe2cve /home/vagrant/rpmbuild/BUILDROOT/nvdtools-1.0-1.x86_64/usr/bin/cpe2cve
+ install -p -m 0755 ./csv2cpe /home/vagrant/rpmbuild/BUILDROOT/nvdtools-1.0-1.x86_64/usr/bin/csv2cpe
+ install -p -m 0755 ./nvdsync /home/vagrant/rpmbuild/BUILDROOT/nvdtools-1.0-1.x86_64/usr/bin/nvdsync
+ install -p -m 0755 ./rpm2cpe /home/vagrant/rpmbuild/BUILDROOT/nvdtools-1.0-1.x86_64/usr/bin/rpm2cpe
+ /usr/lib/rpm/brp-compress
+ /usr/lib/rpm/brp-strip /usr/bin/strip
+ /usr/lib/rpm/brp-strip-static-archive /usr/bin/strip
+ /usr/lib/rpm/brp-strip-comment-note /usr/bin/strip /usr/bin/objdump
Processing files: nvdtools-1.0-1.x86_64
Executing(%license): /bin/sh -e /var/tmp/rpm-tmp.Zo9H2s
+ umask 022
+ cd /home/vagrant/rpmbuild/BUILD
+ cd nvdtools-1.0
+ LICENSEDIR=/home/vagrant/rpmbuild/BUILDROOT/nvdtools-1.0-1.x86_64/usr/share/licenses/nvdtools-1.0
+ export LICENSEDIR
+ /bin/mkdir -p /home/vagrant/rpmbuild/BUILDROOT/nvdtools-1.0-1.x86_64/usr/share/licenses/nvdtools-1.0
+ cp -pr LICENSE /home/vagrant/rpmbuild/BUILDROOT/nvdtools-1.0-1.x86_64/usr/share/licenses/nvdtools-1.0
+ exit 0
Provides: nvdtools = 1.0-1 nvdtools(x86-64) = 1.0-1
Requires(rpmlib): rpmlib(CompressedFileNames) <= 3.0.4-1 rpmlib(PayloadFilesHavePrefix) <= 4.0-1
Requires: libc.so.6()(64bit) libc.so.6(GLIBC_2.2.5)(64bit) libc.so.6(GLIBC_2.3.4)(64bit) libc.so.6(GLIBC_2.4)(64bit) libpthread.so.0()(64bit) libpthread.so.0(GLIBC_2.2.5)(64bit) libpthread.so.0(GLIBC_2.3.2)(64bit)
Checking for unpackaged file(s): /usr/lib/rpm/check-files /home/vagrant/rpmbuild/BUILDROOT/nvdtools-1.0-1.x86_64
Wrote: /home/vagrant/rpmbuild/SRPMS/nvdtools-1.0-1.src.rpm
Wrote: /home/vagrant/rpmbuild/RPMS/x86_64/nvdtools-1.0-1.x86_64.rpm
Executing(%clean): /bin/sh -e /var/tmp/rpm-tmp.dlkDpf
+ umask 022
+ cd /home/vagrant/rpmbuild/BUILD
+ cd nvdtools-1.0
+ /bin/rm -rf /home/vagrant/rpmbuild/BUILDROOT/nvdtools-1.0-1.x86_64
+ exit 0
vagrant@ubuntu-xenial:~/repos/nvdtools$ sudo alien -i /home/vagrant/rpmbuild/RPMS/x86_64/nvdtools-1.0-1.x86_64.rpm
	dpkg --no-force-overwrite -i nvdtools_1.0-2_amd64.deb
(Reading database ... 74500 files and directories currently installed.)
Preparing to unpack nvdtools_1.0-2_amd64.deb ...
Unpacking nvdtools (1.0-2) over (1.0-2) ...
Setting up nvdtools (1.0-2) ...

vagrant@ubuntu-xenial:~/repos/nvdtools$ ls -al /usr/bin/cpe2cve
-rwxr-xr-x 1 root root 2800096 Jan 14 21:09 /usr/bin/cpe2cve
vagrant@ubuntu-xenial:~/repos/nvdtools$ ls -al /usr/bin/csv2cpe
-rwxr-xr-x 1 root root 1769000 Jan 14 21:09 /usr/bin/csv2cpe
vagrant@ubuntu-xenial:~/repos/nvdtools$ ls -al /usr/bin/nvdsync
-rwxr-xr-x 1 root root 6958328 Jan 14 21:09 /usr/bin/nvdsync
vagrant@ubuntu-xenial:~/repos/nvdtools$ ls -al /usr/bin/rpm2cpe
-rwxr-xr-x 1 root root 1764728 Jan 14 21:09 /usr/bin/rpm2cpe
```

### CODE REVIEW

**Comment by skogtwin** [comment]
Hey, thanks for the PR, this is super cool! But if you could amend .travis.yml to make Travis CI build it and publish back to github, that would be plain awesome :) If you're up to it, let's amend this PR, otherwise let me know, I'll merge it and we'll move from there.

File: debian/changelog [hunk 1]
```diff
@@ -0,0 +1,5 @@
+nvdtools (1.0) precise; urgency=low
+
+  * Initial release.
```
Comment (skogtwin) [inline debian/changelog]: I would not call this a release... Feels more like `a .deb package has been added` to me.

File: debian/README.Debian [hunk 2]
```diff
@@ -0,0 +1 @@
+nvdtools for Debian
```
Comment (skogtwin) [inline debian/README.Debian]: I'd rather copy the short description from README.md here. Also, this `for Debian` bit feels a bit confusing to me, like if this project is only available for Debian.

File: debian/changelog (hunk 1 above)
Comment (jokLiu) [reply debian/changelog]: Sure, I will update it.

File: debian/README.Debian (hunk 2 above)
Comment (jokLiu) [reply debian/README.Debian]: That's true. Thanks :)

**Pull request merged**